src/appleseed/renderer/kernel/lighting/pt/ptlightingengine.cpp

```diff
@@ -1161,6 +1167,14 @@ Dictionary PTLightingEngineFactory::get_params_metadata()
             .insert("label", "Next Event Estimation")
             .insert("help", "Explicitly connect path vertices to light sources to improve efficiency"));
 
+    metadata.dictionaries().insert(
+        "clamp_roughness",
+        Dictionary()
+        .insert("type", "bool")
+        .insert("default", "false")
+        .insert("label", "Clamp BSDF roughness")
+        .insert("help", "Clamp BSDF roughness paramater to maximum level to avoid fireflies in glossy reflections"));
```

Comment:
*paramater* -> *parameter*

*to **a** maximum level*

*to avoid* -> *to reduce*